lib/css/components/_stacks-topbar.less

```diff
@@ -0,0 +1,366 @@
+    height: var(--topbar-height);
+    display: flex;
+    border-top: var(--theme-topbar-accent-border);
+    position: relative;
+
+    // TODO necessary? used to wrap content so the topbar stretches 100% w/ content at some value below that
+    .s-topbar--container {
+        width: 100%; // TODO note - consumers should use atomic classes to override this
+        display: flex;
+        margin: 0 auto;
+    }
+
+    .s-topbar--logo {
+        padding: 0 @su8;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        background-color: transparent;
+    }
+
+    a.s-topbar--logo:hover {
+        background-color: var(--theme-topbar-item-background-hover);
+    }
+
+    a.s-topbar--logo.is-selected {
+        background-color: var(--theme-topbar-item-background-hover);
+    }
+
+    .s-topbar--menu {
+        display: flex;// none;
+        padding: 0;
+        height: 100%;
+        width: 44px;
+        flex-shrink: 0;
+        align-items: center;
+        justify-content: center;
+
+        // NOTE: taken from source file
+        // build hamburger icon manually instead of using `@Svg` helper because
+        // we can more easily animate its content (transform hamburger into x)
+        span {
+            width: 18px;
+            height: 2px;
+            background-color: var(--theme-topbar-item-color);
+            position: relative;
+
+            &:before,
+            &:after {
+                position: absolute;
+                content: '';
+                width: 18px;
+                height: 2px;
+                left: 0;
+                background: var(--theme-topbar-item-color);
+                top: -5px;
+                transition: all ease-in-out 0.1s;
+            }
+
+            &:after {
+                top: 5px;
+            }
+        }
+
+        // Transforming hamburger into x
+        &.is-selected {
+            color: var(--theme-topbar-item-color-current);
+            background-color: var(--theme-topbar-item-background-current);
+
+            span {
+                background-color: transparent;
+
+                &:before,
+                &:after {
+                    top: 0;
+                    transform: rotate(-45deg);
+                }
+
+                &:after { transform: rotate(45deg); }
+            }
+        }
+
+        &:hover {
+            color: var(--theme-topbar-item-color-hover);
+            background-color: var(--theme-topbar-item-background-hover);
+        }
+    }
+}
+
+.s-topbar .s-topbar--navigation,
+.s-topbar .s-topbar--content,
+.s-topbar .s-topbar--ctas {
+    display: flex;
+    height: 100%;
+    .list-reset;
+
+    // TODO cleanup
+    > li {
+        display: inline-flex;
+    }
+}
+
+.s-topbar .s-topbar--navigation,
+.s-topbar .s-topbar--content {
+    .s-topbar--item {
+        color: var(--theme-topbar-item-color);
+        display: inline-flex;
+        align-items: center;
+        padding: 0 10px; // TODO @su8 or @su12?
```

Comment:
12, or... if you really wanna be fancy, you could `@su12 - @su2`. We've done worse.

Reply:
Imo, 12 was too much and 8 was too little. I added `(@su12 - @su2)` as suggested so we're at least not _entirely_ hardcoding the value...